Tests/Integration/Network/Protocol/TransactionIdempotenceTests.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+@testable import MobileCoin
+import XCTest
+
+class TransactionIdempotenceTests: XCTestCase {
```

Comment:
Looks like this ended up in the wrong folder in the repo, but also does this need to be an integration test ? seems like its better suited as a unit test case.

Reply:
These are calls that go through to the mobilecoin/libmobilecoin APIs, so it's not really a "unit" test per-se.
Depends on how we categorize tests...

But yes, definitely didn't mean for them to be under Network/Protocol

Reply:
Tests that need network/environment access are Integration tests, if they don't need the network, the default is unit test.

Reply:
Understood.  Moved from Network/Protocol to Transaction/ for now (still under Integration).  Would have to redo a lot of wiring for them to work b/c they're leveraging the existing fixtures for integration tests where the accounts have data / transactions for creating the input and change transaction...  Might not take that long, but I can add a follow up task to move this from integration to unit testing after the code is merged.